user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_ReservationFieldDescriptor.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: DOM_ReservationFieldDescriptor
+---
+
+# ReservationFieldDescriptor
+
+- **FieldValue type**: `GUID` of an SRM `ReservationInstance`
+- **Multiple values optional**: :heavy_check_mark:
+
+| Type of Descriptor | FieldType | FieldValue type |
+|--------------------|-----------|-----------------|
+| Reservation | Guid | Guid |
+| Reservation with multiple values enabled| List\<Guid\> | Guid (ListValueWrapper) |
+
+Defines a field that has the `GUID` of an SRM `(Service)ReservationInstance`.
+
+The `ReservationFieldDescriptor` lists all ReservationInstances, the selected `ReservationInstance` is saved as their `GUID`.
```

Comment:
"the 'GUID' of the selected 'ReservationInstance' is saved" sounds better than "the selected 'ReservationInstance' is saved as their 'GUID'", but this might be preference. Same for the other FieldDescriptors.